### PR TITLE
Fixed escape assertion bug & implemented comment removing

### PR DESCRIPTION
**Pull request by Roman-787**
This is a fix for the escapeAssertion Bug, discussed here: https://github.com/casbin/casbin/issues/192

I also implemented the removeComments Method.

### CODE REVIEW

File: src/main/java/org/casbin/jcasbin/util/Util.java [hunk 1]
```diff
@@ -56,9 +58,23 @@ public static void logPrintf(String format, String... v) {
      * @return the escaped value.
      */
     public static String escapeAssertion(String s) {
-        s = s.replace("r.", "r_");
-        s = s.replace("p.", "p_");
-        return s;
+
```
Comment (hsluoyz) [inline src/main/java/org/casbin/jcasbin/util/Util.java]: Please remove this new line.

File: src/test/java/org/casbin/jcasbin/main/UtilTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import static org.junit.Assert.assertEquals;
+
+import org.casbin.jcasbin.util.Util;
+import org.junit.Test;
+
+public class UtilTest {
+
+  @Test
+  public void testEscapeAssertion(){
+
```
Comment (hsluoyz) [inline src/test/java/org/casbin/jcasbin/main/UtilTest.java]: Please remove this new line.

File: src/test/java/org/casbin/jcasbin/main/UtilTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    assertEquals("r_attp.value <p_attr", Util.escapeAssertion("r.attp.value <p.attr"));
+    assertEquals("r_attp.value -p_attr", Util.escapeAssertion("r.attp.value -p.attr"));
+    assertEquals("r_attp.value +p_attr", Util.escapeAssertion("r.attp.value +p.attr"));
+    assertEquals("r_attp.value *p_attr", Util.escapeAssertion("r.attp.value *p.attr"));
+    assertEquals("r_attp.value /p_attr", Util.escapeAssertion("r.attp.value /p.attr"));
+    assertEquals("!r_attp.value /p_attr", Util.escapeAssertion("!r.attp.value /p.attr"));
+    assertEquals("g(r_sub, p_sub) == p_attr", Util.escapeAssertion("g(r.sub, p.sub) == p.attr"));
+    assertEquals("g(r_sub,p_sub) == p_attr", Util.escapeAssertion("g(r.sub,p.sub) == p.attr"));
+    assertEquals("(r_attp.value || p_attr)p_u", Util.escapeAssertion("(r.attp.value || p.attr)p.u"));
+
```
Comment (hsluoyz) [inline src/test/java/org/casbin/jcasbin/main/UtilTest.java]: Please remove this new line.

File: src/test/java/org/casbin/jcasbin/main/UtilTest.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  @Test
+  public void testRemoveComments(){
+
```
Comment (hsluoyz) [inline src/test/java/org/casbin/jcasbin/main/UtilTest.java]: Please remove this new line.

File: src/test/java/org/casbin/jcasbin/main/UtilTest.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    assertEquals("r.act == p.act", Util.removeComments("r.act == p.act # comments"));
+    assertEquals("r.act == p.act", Util.removeComments("r.act == p.act#comments"));
+    assertEquals("r.act == p.act", Util.removeComments("r.act == p.act###"));
+    assertEquals("", Util.removeComments("### comments"));
+    assertEquals("r.act == p.act", Util.removeComments("r.act == p.act"));
+
```
Comment (hsluoyz) [inline src/test/java/org/casbin/jcasbin/main/UtilTest.java]: Please remove this new line.

File: src/test/java/org/casbin/jcasbin/main/UtilTest.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
```
Comment (hsluoyz) [inline src/test/java/org/casbin/jcasbin/main/UtilTest.java]: Please remove this new line.

**Comment by Roman-787** [comment]
I looked into other codeblocks and adapted the lines and style.

**Comment by hsluoyz** [comment]
lgtm. Thanks for the contribution!

**Pull request merged**